packages/neon/neon/lib/src/widgets/account_selection_dialog.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import 'package:meta/meta.dart';
+import 'package:neon/src/blocs/accounts.dart';
+import 'package:neon/src/theme/dialog.dart';
+import 'package:neon/src/widgets/account_tile.dart';
+import 'package:provider/provider.dart';
+
+@internal
+class NeonAccountSelectionDialog extends StatelessWidget {
+  const NeonAccountSelectionDialog({
+    this.highlightActiveAccount = false,
+    this.children,
+    super.key,
+  });
+
+  final bool highlightActiveAccount;
+  final List<Widget>? children;
+
+  @override
+  Widget build(final BuildContext context) {
+    final accountsBloc = Provider.of<AccountsBloc>(context, listen: false);
+    final accounts = accountsBloc.accounts.value;
+    final activeAccount = accountsBloc.activeAccount.value!;
+
+    final inactiveAccounts = List.of(accounts)..removeWhere((final account) => account.id == activeAccount.id);
+
+    final tiles = [activeAccount, ...inactiveAccounts].map(
+      (final account) {
+        final highlight = highlightActiveAccount && account.id == activeAccount.id;
+        Widget widget = NeonAccountTile(
+          account: account,
+          trailing: highlight ? const Icon(Icons.check_circle) : null,
+          onTap: () {
+            Navigator.of(context).pop(account);
+          },
+        );
+        if (highlight) {
+          widget = Column(
+            children: [
+              widget,
+              const Divider(),
+            ],
+          );
```

Comment:
I don't like nested columns.

Maybe jsut call `tiles.insert(1, const Divider())` after the map.